src/Controls/src/Core/HandlerImpl/RadioButton/RadioButton.Tizen.cs

```diff
@@ -76,6 +75,55 @@ static View BuildTizenDefaultTemplate()
 				VerticalOptions = LayoutOptions.Center
 			};
 
+			object outerEllipseVisualStateLight;
+			object outerEllipseVisualStateDark;
+			object checkMarkVisualStateLight;
+			object checkMarkVisualStateDark;
+
+			// First, Light/Dark App themes for outer ellipse, then for the check mark.
+			// Then check for older themecolor.
+			// If nothing set, use the defaults for light/dark mode.
+			if (Application.Current.TryGetResource(RadioButtonOuterEllipseStrokeLight, out var outerLight) &&
+				Application.Current.TryGetResource(RadioButtonOuterEllipseStrokeDark, out var outerDark))
+			{
+				normalEllipse.SetAppTheme(Ellipse.StrokeProperty, outerLight, outerDark);
+				outerEllipseVisualStateLight = outerLight;
+				outerEllipseVisualStateDark = outerDark;
+			}
+			else if (Application.Current.TryGetResource(RadioButtonThemeColor, out var themeColor))
+			{
+				normalEllipse.SetDynamicResource(Ellipse.StrokeProperty, RadioButtonThemeColor);
+				outerEllipseVisualStateLight = outerEllipseVisualStateDark = themeColor;
+			}
+			else
+			{
+				normalEllipse.SetAppTheme(Ellipse.StrokeProperty, SolidColorBrush.Black, SolidColorBrush.White);
+				outerEllipseVisualStateLight = SolidColorBrush.Black;
+				outerEllipseVisualStateDark = SolidColorBrush.White;
+			}
+
+			if (Application.Current.TryGetResource(RadioButtonCheckGlyphStrokeLight, out var checkLight) &&
+				Application.Current.TryGetResource(RadioButtonCheckGlyphStrokeDark, out var checkDark))
+			{
+				checkMark.SetAppTheme(Ellipse.StrokeProperty, checkLight, checkDark);
+				checkMark.Fill = (Brush)(Application.Current?.RequestedTheme == AppTheme.Dark ? checkDark : checkLight);
```

Comment:
should we just go all in here on the keys? 

Create a `RadioButtonCheckGlyphFill` for this fill color?